integration_tests/mockApis/prison.ts

```diff
@@ -276,7 +276,7 @@ export default {
         headers: {
           'Content-Type': 'application/json;charset=UTF-8',
         },
-        jsonBody: mockAddresses,
+        jsonBody: resp !== undefined ? resp : mockAddresses,
       },
     })
   },
```

Comment:
I think these are neater with default args like [here](https://github.com/ministryofjustice/hmpps-prisoner-profile/blob/f8afab9e9528bc2575689371575708978bf2b87b/integration_tests/mockApis/prison.ts#L1355-L1375), but there are a number of each style atm